src/templates/common/phoneNumber/index.tsx

```diff
@@ -84,19 +95,32 @@ export const PhoneNumber = ({
 
   const internationalPattern = /\(?(\+1)\)?[- ]?/gi
 
+  const phoneComponent = (
+    <va-telephone
+      contact={numberToDisplay.replace?.(/-/g, '')}
+      extension={extensionToDisplay || null}
+      message-aria-describedby={isRegularPhone ? label || 'Phone' : undefined}
+      not-clickable={fax ? true : undefined}
+      sms={sms ? true : undefined}
+      tty={tty ? true : undefined}
+      international={internationalPattern.test(numberToDisplay)}
+      data-testid={testId || undefined}
+    ></va-telephone>
```

Comment:
these non-self closing tags still bug me. But this is the way we have it everywhere and how it was before. I'll get over it.